Dockerfile

```diff
@@ -58,14 +57,12 @@ RUN cd /tmp \
     && rm -rf /tmp/bats-core
 
 # install criu
-ARG CRIU_VERSION
-RUN mkdir -p /usr/src/criu \
-    && curl -fsSL https://github.com/checkpoint-restore/criu/archive/${CRIU_VERSION}.tar.gz | tar -C /usr/src/criu/ -xz --strip-components=1 \
-    && cd /usr/src/criu \
-    && echo 1 > .gitid \
-    && make -j $(nproc) install-criu \
-    && cd - \
-    && rm -rf /usr/src/criu
+RUN echo 'deb http://ppa.launchpad.net/criu/ppa/ubuntu/ bionic main' > /etc/apt/sources.list.d/criu.list \
+    && wget -nv 'https://keyserver.ubuntu.com/pks/lookup?op=get&search=0x4e2a48715c45aeec077b48169b29eec9246b6ce2' -O- | apt-key add - \
```

Comment:
Why aren't you using something like 'add-apt-repository -y ppa:criu/ppa'?

Reply:
I tried it and it did not work. Apparently this util is Ubuntu specific, and this Dockerfile is Debian based.

Reply:
Right. These tests are running in a container. I expected this to be the travis environment but it is not. This means CRIU should also provide a binary for Debian. Because right now it is basically luck that the Ubuntu package runs on Debian.

Reply:
> right now it is basically luck that the Ubuntu package runs on Debian

Well I assumed since Ubuntu releases are based on Debian releases, they are compatible.

I googled and apparently I was wrong, so yes, it works only by a coincidence.

Now, official golang docker images (https://hub.docker.com/_/golang) are either Debian or Alpine based. We can obviously switch to Ubuntu and install golang from the binaries, but I'm not 100% convinced it makes sense to do it only because of criu.

Reply:
@adrianreber OK I think having a Debian repo for CRIU would also benefit at least Moby (that currently also compiles CRIU from sources).

Reply:
I am looking how to create and host Debian packages for CRIU. I will let you know. Right now I am looking at https://build.opensuse.org/ which could help.

Reply:
Are those repos x86 only, or also other arches?

Reply:
> Are those repos x86 only, or also other arches?

The CRIU ppa provides packages for aarch64, ppc64le, s390x and x86_64.

Although Travis also provides all those arches CRIU only works on x86_64 as the other Travis arches are running in LXD containers which are too unprivileged for CRIU.

Reply:
@kolyshkin please try the packages at https://download.opensuse.org/repositories/devel:/tools:/criu/Debian_10/

Reply:
@adrianreber thanks! Looks like opensuse.org is down ATM :(

Reply:
OK it's back, PR updated